enterprise/app/trends/cache_chart.tsx

```diff
@@ -6,29 +6,33 @@ import { invocation } from "../../../proto/invocation_ts_proto";
 interface Props {
   title: string;
   data: any[];
+  secondaryBarName: string;
   extractLabel: (datum: any) => string;
   formatHoverLabel: (datum: any) => string;
   extractHits: (datum: any) => number;
-  extractMisses?: (datum: any) => number;
-  extractWrites?: (datum: any) => number;
+  extractSecondary: (datum: any) => number;
 }
 
-const CacheChartTooltip = ({ active, payload, labelFormatter, extractHits, extractMisses, extractWrites }: any) => {
+const CacheChartTooltip = ({
+  active,
+  payload,
+  labelFormatter,
+  extractHits,
+  secondaryBarName,
+  extractSecondary,
+}: any) => {
```

Comment:
nit, do you mind defining a `CacheChartTooltipProps` while you're here?

Reply:
Sorry for sitting on this, here are all the `TooltipProps` you could ever want!